plugins/modules/vmware_vcenter_settings_info.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, sky-joker
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+module: vmware_vcenter_settings_info
+short_description: Gather info vCenter settings
+description:
+  - This module can be used to gather information about vCenter settings.
+author:
+  - sky-joker (@sky-joker)
+requirements:
+  - python >= 2.7
+  - PyVmomi
+extends_documentation_fragment:
+  - community.vmware.vmware.documentation
+'''
+
+EXAMPLES = '''
+- name: "Gather info about vCenter settings"
+  vmware_vcenter_settings_info:
+    hostname: "{{ vcenter_hostname }}"
+    username: "{{ vcenter_username }}"
+    password: "{{ vcenter_password }}"
+    validate_certs: no
+  register: vcenter_settings_info
+'''
+
+RETURN = '''
+result:
```

Comment:
`result` is internal to Ansible Core engine, please change it something relevant as `vcenter_config_info` or `vcenter_info`